### PR TITLE
Icon as button update - adding transparent version to action buttons.

### PR DESCRIPTION
**Pull request by burakukula**
icons-update

### CODE REVIEW

File: src/components/buttons/_icon-as-button.scss [hunk 1]
```diff
@@ -151,6 +151,7 @@ $includeHtml: false !default;
       border-radius: 50%;
 
       &:hover {
+        fill: $iconAsButtonLightColor;
```
Comment (aju) [inline src/components/buttons/_icon-as-button.scss]: why we need it?

**Comment by aju** [comment]
@burakukula I would present it as a group (look at `Action` and `Action + square`) @kdzwinel WDYT ?

File: src/components/buttons/_icon-as-button.scss (hunk 1 above)
Comment (burakukula) [inline src/components/buttons/_icon-as-button.scss]: @aju `--light` overwrites this fill and we need it white on hover.

**Comment by kdzwinel** [comment]
if we treat it as a color it should be everywhere, if we treat it as a type it should be in a separate group. TBH IDK how to think about this one

File: src/components/buttons/_icon-as-button.scss (hunk 1 above)
Comment (aju) [inline src/components/buttons/_icon-as-button.scss]: Then we need to do it like `&--action {` where blue is our main color

**Pull request merged**